packages/paste-website/src/components/AspectRatio.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {Absolute} from '@twilio-paste/absolute';
+import {Box} from '@twilio-paste/box';
+
+interface AspectRatioProps {
+  aspectRatio: string;
+  children: React.ReactElement;
+}
+
+const RATIO_REGEX = /^(\d+:\d*)$/;
+
+const isCorrectPattern = (ratio: string): boolean => RATIO_REGEX.test(ratio);
+
+const handlePropValidation = ({aspectRatio, children}: AspectRatioProps): void => {
+  const hasRatio = aspectRatio != null && aspectRatio !== '';
+
+  if (!hasRatio) {
+    throw new Error(`[Paste: AspectRatio] Missing hasRatio prop for AspectRatio.`);
+  }
+
+  if (!isCorrectPattern(aspectRatio)) {
+    throw new Error(
+      `[Paste: AspectRatio] hasRatio prop for AspectRatio is incorrect. Use a colon separated number pattern like 4:3.`
```

Comment:
I think you meant `[Paste: AspectRatio] 'aspectRatio' is invalid. Use a colon separated number pattern (4:3).`

Reply:
Yep. Mis-typed that one.